### DISCUSSION TITLE
Support for Badge and Icon

### DISCUSSION THREAD

**Discussion by ManojKiranA** [filamentphp/filament#5445] [Ideas]
After discussion with @danharrin on [discord](https://discord.com/channels/883083792112300104/883084832387760148/1062252135821099008) for adding Badge and Icon to `Relation Manager` tabs and `List Page` tabs

Something similar to this `API` would be nice

For List Page Query tabs

```
/**
* @return array<string, Closure | null>
*/
public function getQueryTabs(): array
{
   return [
      Tab::make('banned_users')
          ->label('Banned')
          ->icon('heroicon-o-ban')
          ->iconPosition('before')
          ->badge('10')
          ->query(...),

     Tab::make('active_users')
          ->label('Active')
          ->icon('heroicon-o-check-circle')
          ->iconPosition('before')
          ->badge('70')
          ->query(...),
  ];
}
```

**Comment by danharrin** [comment]
Implemented in v3

**Reply by ManojKiranA** [reply]
Wow that was fast. Thanks @danharrin

**Comment by ManojKiranA** [comment]
This is implemented and you can find the docs here [here](https://filamentphp.com/docs/3.x/panels/resources/listing-records#using-tabs-to-filter-the-records)
